charts/osm/values.schema.json

```diff
@@ -265,20 +267,29 @@
                                 "IfNotPresent"
                             ]
                         },
-                        "defaultOutput": {
-                            "$id": "#/properties/OpenServiceMesh/properties/fluentBit/properties/defaultOutput",
+                        "outputName": {
+                            "$id": "#/properties/OpenServiceMesh/properties/fluentBit/properties/outputName",
                             "type": "string",
-                            "title": "The default output schema for Fluent Bit",
-                            "description": "The default log forwarding destination for Fluent Bit",
+                            "title": "The output schema for Fluent Bit",
+                            "description": "The log forwarding destination for Fluent Bit",
                             "examples": [
                                 "stdout"
                             ]
                         },
-                        "allowCustomOutput": {
-                            "$id": "#/properties/OpenServiceMesh/properties/fluentBit/properties/allowCustomOutput",
-                            "type": "boolean",
-                            "title": "The allowCustomOutput output schema",
-                            "description": "Indicates whether users can provide their own output plugin",
+                        "workspaceId": {
+                            "$id": "#/properties/OpenServiceMesh/properties/fluentBit/properties/workspaceId",
+                            "type": "string",
+                            "title": "The Log Analytics workspaceId schema",
```

Comment:
If this refers specifically to Azure Log Analytics, then we should include that. People not familiar with Log Analytics may get confused otherwise.

Reply:
Yeah that makes sense!